src/main/java/net/greatstart/mappers/InvestmentMapper.java

```diff
@@ -0,0 +1,18 @@
+package net.greatstart.mappers;
+
+import net.greatstart.dto.DtoInvestment;
+import net.greatstart.model.Investment;
+import org.mapstruct.Context;
+import org.mapstruct.Mapper;
+import org.mapstruct.factory.Mappers;
+
+@Mapper(componentModel = "spring")
+public interface InvestmentMapper {
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Move constants to a class or enum. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS1214)